opal_in_production.org

```diff
@@ -1,81 +1,92 @@
 One thing I believe thats hinders [[http://opalrb.org][Opal's]] adoption is that people are
 unsure whether Opal is "ready for production." On May 8, 2015, we
 deployed an Opal (and Rails) app to production.  While I've done a
-fair amount of work with Opal in the last few years, open source, side
-projects, internal tools (one of which could become a public facing
-tool), it was only on May 8th that I joined the ranks of those having
-a production site using Opal.
```

Comment:
Dropped what you've done w/ Opal:

1.  Not relevant to what you're trying to get across.
1.  I can't find a way to not make the sentence structure awkward.